build.cake

```diff
@@ -115,6 +115,17 @@ Task("Test")
         .Where(name => !name.ToString().Contains("Atlas")),
         testProject => 
     {
+        var testWithDefaultGuidRepresentationMode = Environment.GetEnvironmentVariable("TEST_WITH_DEFAULT_GUID_REPRESENTATION_MODE");
+        if (testWithDefaultGuidRepresentationMode != null)
+        {
+            Console.WriteLine($"TEST_WITH_DEFAULT_GUID_REPRESENTATION_MODE={testWithDefaultGuidRepresentationMode}");
+        }
+        var testWithDefaultGuidRepresentation = Environment.GetEnvironmentVariable("TEST_WITH_DEFAULT_GUID_REPRESENTATION");
+        if (testWithDefaultGuidRepresentation != null)
+        {
+            Console.WriteLine($"TEST_WITH_DEFAULT_GUID_REPRESENTATION={testWithDefaultGuidRepresentation}");
+        }
+
```

Comment:
This change to build.cake doesn't affect anything. It only echoes to the console the value of a couple of environment variable (if defined).